src/status_im/data_store/realm/schemas/base/migrations.cljs

```diff
@@ -130,3 +131,42 @@
 
 (defn v18 [old-realm new-realm]
   (log/debug "migrating base database v18: " old-realm new-realm))
+
+;; transform inactive legacy network values into the current ones
+(defn- transition-rpc-url [rpc-url]
+  (cond
+    (= rpc-url "https://mainnet.infura.io/z6GCTmjdP3FETEJmMBI4")
+    (get-in constants/mainnet-networks ["mainnet_rpc" :config :UpstreamConfig :URL])
+    (= rpc-url "https://ropsten.infura.io/z6GCTmjdP3FETEJmMBI4")
+    (get-in constants/testnet-networks ["testnet_rpc" :config :UpstreamConfig :URL])
+    (= rpc-url "https://rinkeby.infura.io/z6GCTmjdP3FETEJmMBI4")
+    (get-in constants/testnet-networks ["rinkeby_rpc" :config :UpstreamConfig :URL])
+    :else rpc-url))
+
+(defn- update-infura-project-id! [network-js]
+  (let [old-config (js->clj
+                    (.parse js/JSON
+                            (aget network-js "config")))]
+    ;; we only transition rpc networks
+    (when (get-in old-config ["UpstreamConfig" "Enabled"] false)
```

Comment:
Not sure you need to default to false (`nil` is a falsey value)

Reply:
done